index.html

```diff
@@ -0,0 +1,71 @@
+<!-- Este es un comentario -->
+<!-- Estructura básica del documento HTML -->
+<!DOCTYPE html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mi CV</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<header>
+    <img src="juanjoImagen.jpg" alt="juanjoImagen"> 
+    <div class="DivEncabezado">
+        <div class="divh1">
+            <h1>
+                 Juan José Gómez Saavedra
+            </h1>
+        </div>
+        <div class="informacionpersonal">
```

Comment:
Debemos programar en inglés

```suggestion
        <div class="personal-information">
```